lib/s3routes/routesUtils.js

```diff
@@ -8,6 +8,13 @@ const responseErr = new Error();
 responseErr.code = 'ResponseError';
 responseErr.message = 'response closed by client request before all data sent';
 
+// Setting a lower object key limit to account for:
+// - Mongo key limit of 1012
```

Comment:
May be worth specifying that this limit is a byte limit of the string rather than a character length limit, unless I'm mistaken about this.